idutils/proxies.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of IDUtils
+# Copyright (C) 2024 CERN.
+#
+# IDUtils is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Proxy definitions."""
+
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+current_idutils = LocalProxy(lambda: current_app.extensions["idutils"])
```

Comment:
I think renaming this to something that includes `scheme` and `registry` might be better.

Reply:
I matched to the convention we are following e.g.: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/proxies.py#L14